### PR TITLE
Don't submit TOTP input values without user action

### LINKED ISSUE #77: QR code notice displays when saving FIDO as the primary provider
> Two Factor Authentication not activated, you must specify authcode to ensure it is properly set up. Please re-scan the QR code and enter the code provided by your application.

### PR DESCRIPTION
**Pull request by shield-9**
This fixes #77

By disabling input element, its value won't be submitted to the server.

### CODE REVIEW

**Comment by aaroncampbell** [comment]
I haven't tested or anything, but it seems like this would break no-js, which feels like a regression.

**Comment by valendesigns** [comment]
This PR brought to my attention the fact we have inline JS in the plugin, which needs to be written differently. Inline `onclick` events like this are not allowed in Core.

**Comment by valendesigns** [comment]
This PR needs to be updated to not use a JS approach. It should probably attempt to filter out the input value when the form is submitted instead.

File: providers/class.two-factor-totp.php [hunk 1]
```diff
@@ -99,7 +100,7 @@ public function user_two_factor_options( $user ) {
 	 * @param integer $user_id The user ID whose options are being updated.
 	 */
 	public function user_two_factor_options_update( $user_id ) {
-		if ( isset( $_POST['_nonce_user_two_factor_totp_options'] ) ) {
+		if ( ! empty( $_POST['do_totp_authcode'] ) ) {
```
Comment (valendesigns) [inline providers/class.two-factor-totp.php]: This should probably still check for the nonces existence. For example:
`if ( ! empty( $_POST['do_totp_authcode'] ) && isset( $_POST['_nonce_user_two_factor_totp_options'] ) ) {`

**Comment by georgestephanis** [comment]
I believe this was fixed by #145 so we don't need this more complex fix?  Happy to reopen if I'm mistaken.

**Pull request closed without merging**